### PR TITLE
feat: improve NftCardGrid

### PR DESCRIPTION
**Pull request by sk-umiuma**
https://user-images.githubusercontent.com/110230002/227104015-ba415826-9a19-4f25-948c-4c551a3141f9.mp4

Resolves: https://app.asana.com/0/0/1204231410171179/f

### CODE REVIEW

**Comment by sk-umiuma** [comment]
Uploading Screen Recording 2023-03-23 at 11.25.41.mov…

File: containers/PageKolours/containers/NftCard/components/CompleteIndicator/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.container {
+  background-color: white;
+  border-radius: 50%;
+}
+
+.percentage {
+  position: absolute;
+  top: 0;
+  width: 40px;
+  height: 40px;
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/NftCard/components/CompleteIndicator/index.module.scss:10]: When using `position: absolute`, I recommend define **4 out of 6** properties: left, right, top, bottom, width, height

You have defined 3, you should add one more (`left` or `right`)

File: containers/PageKolours/containers/NftCard/index.tsx [hunk 2]
```diff
@@ -19,15 +22,19 @@ type Props = {
 export default function NftCard({ className, style, value, onClick }: Props) {
   return (
     <div className={cx(styles.container, className)} style={style}>
-      <button onClick={onClick} className={styles.button}>
-        <WithAspectRatio aspectRatio={2 / 1}>
+      <div className={styles.box} onClick={onClick}>
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/NftCard/index.tsx:25]: When we have a `<div>` with `onClick`, most of the time, we should either use `<button>` or `<a>`

This should be a button.

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Pull request merged**